src/components/home/components/styles.css

```diff
@@ -173,6 +185,8 @@ img {
 .link-position {
     float: right;
     margin-top: 4px;
+    margin-right: 600px;
+    width: 485px;
```

Comment:
use `float: left` instead of `margin-right`

Reply:
float-right not working